app/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source.rb

```diff
@@ -47,6 +47,7 @@ def sync
   #   https://github.ibm.com/manoj-puthran/sample-scripts/tree/v2.0/terraform/templates/hello-world
   #       is converted as
   #   "hello-world(v2.0):github.ibm.com/manoj-puthran/sample-scripts/terraform/templates"
```

Comment:
I think we can delete this comment since it's the old way

Reply:
Fixed and repushed